docs/azure/ai/get-started-app-chat-template.md

```diff
@@ -0,0 +1,300 @@
+* An Azure OpenAI Service to provide:
+  * Keywords to enhance the search over your own data.
+  * Answers from the OpenAI model.
+  * Embeddings from the ada model
+
+## Cost
+
+Most resources in this architecture use a basic or consumption pricing tier. Consumption pricing is based on usage, which means you only pay for what you use. To complete this article, there will be a charge but it will be minimal. When you are done with the article, you can delete the resources to stop incurring charges.
+
+Learn more about [cost in the sample repo](https://github.com/Azure-Samples/azure-search-openai-demo-csharp#cost-estimation).
+
+## Prerequisites
+
+A [development container](https://containers.dev/) environment is available with all dependencies required to complete this article. You can run the development container in GitHub Codespaces (in a browser) or locally using Visual Studio Code.
+
+To use this article, you need the following prerequisites:
+
+#### [Codespaces (recommended)](#tab/github-codespaces)
+
+1. An Azure subscription - [Create one for free](https://azure.microsoft.com/free/cognitive-services?azure-portal=true)
+1. GitHub account
+
+#### [Visual Studio Code](#tab/visual-studio-code)
+
+1. An Azure subscription - [Create one for free](https://azure.microsoft.com/free/cognitive-services?azure-portal=true)
+1. [Azure Developer CLI](/azure/developer/azure-developer-cli/install-azd?tabs=winget-windows%2Cbrew-mac%2Cscript-linux&pivots=os-windows)
+1. [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+1. [Visual Studio Code](https://code.visualstudio.com/)
+1. [Dev Container Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+---
+
+## Open development environment
+
+Begin now with a development environment that has all the dependencies installed to complete this article.
+
+#### [GitHub Codespaces (recommended)](#tab/github-codespaces)
+
+[GitHub Codespaces](https://docs.github.com/codespaces) runs a development container managed by GitHub with [Visual Studio Code for the Web](https://code.visualstudio.com/docs/editor/vscode-web) as the user interface. For the most straightforward development environment, use GitHub Codespaces so that you have the correct developer tools and dependencies preinstalled to complete this article.
+
+> [!IMPORTANT]
+> All GitHub accounts can use Codespaces for up to 60 hours free each month with 2 core instances. For more information, see [GitHub Codespaces monthly included storage and core hours](https://docs.github.com/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#monthly-included-storage-and-core-hours-for-personal-accounts).
+
+1. Start the process to create a new GitHub Codespace on the `main` branch of the [`Azure-Samples/azure-search-openai-demo-csharp`](https://github.com/Azure-Samples/azure-search-openai-demo-csharp) GitHub repository:
```

Comment:
Back ticks don't render and can sometimes cause issues when placed within a link like this.

```suggestion
1. Start the process to create a new GitHub Codespace on the `main` branch of the [Azure-Samples/azure-search-openai-demo-csharp](https://github.com/Azure-Samples/azure-search-openai-demo-csharp) GitHub repository:
```